runtime/oti/VMHelpers.hpp

```diff
@@ -2048,18 +2049,56 @@ class VM_VMHelpers
 #endif /* JAVA_SPEC_VERSION > 11 */
 	}
 
-	static VMINLINE UDATA
-	walkContinuationStackFramesWrapper(J9VMThread *vmThread, j9object_t continuationObject, J9StackWalkState *walkState)
-	{
-		UDATA rc = J9_STACKWALK_RC_NONE;
 #if JAVA_SPEC_VERSION >= 19
-		J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, continuationObject);
-		rc = vmThread->javaVM->internalVMFunctions->walkContinuationStackFrames(vmThread, continuation, walkState);
-#endif /* JAVA_SPEC_VERSION >= 19 */
-		return rc;
+/*
+ *	If low tagging failed (or not even attempted) due to either
```

Comment:
if we collapse the method to always attempting the atomic, then we can remove '(or not even attempted)' part of the comment